resource/definitions/loader/3dsmax/rig-3dsmax-loader.json

```diff
@@ -0,0 +1,207 @@
+{
+  "type": "loader",
+  "name": "Rig Loader",
+  "asset_type": "rig",
+  "host_type": "3dsmax",
+  "ui_type": "qt",
+  "contexts": [
+    {
+      "name": "main",
+      "stages": [
+        {
+          "name": "context",
+          "plugins":[
+            {
+              "name": "context selector",
+              "plugin": "common_passthrough_loader_context",
+              "widget": "common_default_loader_context"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "components": [
+    {
+      "name": "snapshot",
+      "file_formats": [".max"],
+      "stages": [
+        {
+          "name": "collector",
+          "plugins":[
+            {
+              "name": "Collect components from context",
+              "plugin": "common_context_loader_collector"
+            }
+          ]
+        },
+        {
+          "name": "importer",
+          "plugins":[
+            {
+              "name": "Import paths to Max",
+              "plugin": "max_native_loader_importer",
+              "options": {
+                "load_mode": "import"
+              }
+            }
+          ]
+        },
+        {
+          "name": "post_importer",
+          "plugins":[
+            {
+              "name": "max",
+              "plugin": "common_passthrough_loader_post_importer"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "model",
+      "file_formats": [".max"],
+      "stages": [
+        {
+          "name": "collector",
+          "plugins":[
+            {
+              "name": "Collect components from context",
```

Comment:
@henriknorin-ftrack 

`Collect components from context`

is it what does this do ?

Reply:
# Plugin that collects loader compatible component paths

I'll change to something like this